### PR TITLE
Contribution branch

### PR DESCRIPTION
**Pull request by BatoolMM**
I added CONTRIBUTING.md file. 
Please let me know if you need me to change/add something.

### CODE REVIEW

**Comment by BatoolMM** [comment]
Can you review this pull request, please?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Contributing Guidelines:
+
+We welcome all contributions to the CapShun project. The contribution can be a [issue report](https://github.com/devcer/CapShun/issues) 
+or a [pull request](https://github.com/devcer/CapShun/pulls)..
```
Comment (devcer) [inline CONTRIBUTING.md:4]: remove two dots.

**Comment by BatoolMM** [comment]
removed as requested.

**Comment by devcer** [comment]
@BatoolMM can you add instructions to clone, fork, checkout to the same contribution file? You can find similar changes in this PR -> https://github.com/devcer/hydrator/pull/10

**Comment by BatoolMM** [comment]
I added it.

**Comment by BatoolMM** [comment]
Can you review this pull request, please?

**Comment by devcer** [comment]
@BatoolMM I requested you to add additional information. not to remove the one which you already added. This is not GATK project as you mentioned in the markdown file. Please fix it to this can be reviewed.

**Comment by BatoolMM** [comment]
@devcer I didn't remove anything, I only added additional information.

**Review by devcer — APPROVED** [review]

**Comment by devcer** [comment]
Looks good! Happy Hacktoberfest.

**Pull request merged**